docs/learn/advanced/privacy.md

```diff
@@ -0,0 +1,20 @@
+# Privacy
+This page describes various privacy aspects of the Ceramic protocol. Measures we've taken so far, planned future measures, and future research directions.
+
+## Ceramic Protocol
+The StreamType system is already quite flexible in terms of how streams can provide developers with privacy features.
+
+### Default properties of Tile and Caip10Links
+Currently if you create a TileDocument or Caip10Link stream any data put into it will be public by default. It's however possible to encrypt the content put into the stream by accessing the DID instance from Ceramic (`ceramic.did`) and using it's encryption functionality as described in the [How to store signed and encrypted data on IPFS](https://blog.ceramic.network/how-to-store-signed-and-encrypted-data-on-ipfs/) blog post.
+
+### Confidential streams
+A planned improvement to the TileDocument StreamType is to add confidentiality. This basically means that the content of each update to the stream is encrypted by a symmetric key. Whenever a Ceramic node syncs the stream it would only be able to read the stream content if it has the symmetric key for this stream. Stream metadata such as which DID signed the update, in what order, and when it was anchored would still be public.
```

Comment:
```suggestion
A planned improvement to the TileDocument StreamType is to add confidentiality. This basically means that the content of each update to the stream would be encrypted by a symmetric key. Whenever a Ceramic node syncs the stream it would only be able to read the stream content if it has the symmetric key for this stream. Stream metadata such as which DID signed the update, in what order, and when it was anchored would still be public.
```